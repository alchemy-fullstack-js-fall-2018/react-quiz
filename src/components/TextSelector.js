import React from 'react';

export default function TextSelector({
  text,
  onChange
}) {
  return (
    <div>
      <label name="text">Text:</label>
      <input
        name="text"
        type="text"
        value={text}
        onChange={onChange}
      />
    </div>
  );
}
