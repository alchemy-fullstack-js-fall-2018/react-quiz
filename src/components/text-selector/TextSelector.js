import React, { Fragment } from 'react';

const TextSelector = ({ text, handleChange }) => {
  return (
    <Fragment>
      <input  name="text" value={text} onChange={handleChange}/>
    </Fragment>
  );
};

export default TextSelector;

