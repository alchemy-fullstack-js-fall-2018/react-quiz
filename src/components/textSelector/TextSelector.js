import React, { Fragment } from 'react';
import PropTypes from 'prop-types';

function TextSelector({ artist, works }) {

  return (
    <Fragment>
      <input
        type="text"
      ></input>

    </Fragment>
  );
}

TextSelector.propTypes = {
  // note: PropTypes.object.isRequired
};

export default TextSelector;
