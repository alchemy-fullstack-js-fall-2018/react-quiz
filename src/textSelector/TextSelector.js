import React from 'react';

const TextSelector = (body) => {
  const container = body;
  return (
    <label>Enter Text here</label>
    <input>{container}</input>
  );
}

export default TextSelector;
