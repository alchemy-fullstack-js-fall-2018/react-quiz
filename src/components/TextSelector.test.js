import React from 'react';
import TextSelector from './TextSelector';
import render from 'react-test-renderer';

describe('TextSelector Component Test', () => {
  test('TextSelector snapshot', () => {
    expect(render
      .create(<TextSelector />)
      .toJSON()).toMatchSnapshot();
  });
});
