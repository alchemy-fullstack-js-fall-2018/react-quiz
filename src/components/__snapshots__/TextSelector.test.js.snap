// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextSelector Component Test TextSelector snapshot 1`] = `
<div>
  <label
    name="text"
  >
    Text:
  </label>
  <input
    name="text"
    type="text"
  />
</div>
`;
